From 05ebdd992e96cb31254e0e2ed102be3acebd22b3 Mon Sep 17 00:00:00 2001
From: Joseph Myers <joseph@codesourcery.com>
Date: Tue, 9 Aug 2016 14:18:42 -0700
Subject: [PATCH 02/16] Merge "VFP ABI dynamic linker compatibility" release
 note (CB-1356).

	Merge from SCB glibc 2.23:

	2016-02-27  Sandra Loosemore  <sandra@codesourcery.com>

	CB-1356

	* release-notes-csl.xml (VFP ABI dynamic linker compatibility):
	New note.
---
 ChangeLog.csl         | 11 +++++++++++
 release-notes-csl.xml | 17 +++++++++++++++++
 2 files changed, 28 insertions(+)

diff --git a/ChangeLog.csl b/ChangeLog.csl
index 440335a..cc98729 100644
--- a/ChangeLog.csl
+++ b/ChangeLog.csl
@@ -1,5 +1,16 @@
 2016-08-09  Joseph Myers  <joseph@codesourcery.com>
 
+	Merge from SCB glibc 2.23:
+
+	2016-02-27  Sandra Loosemore  <sandra@codesourcery.com>
+
+	CB-1356
+
+	* release-notes-csl.xml (VFP ABI dynamic linker compatibility):
+	New note.
+
+2016-08-09  Joseph Myers  <joseph@codesourcery.com>
+
 	* release-notes-csl.xml (GNU C Library version 2.24): New release
 	note.
 
diff --git a/release-notes-csl.xml b/release-notes-csl.xml
index 66077bd..e8bc0cd 100644
--- a/release-notes-csl.xml
+++ b/release-notes-csl.xml
@@ -5,6 +5,23 @@
      relevant; it may be removed if it is relevant to all targets.
  -->
 
+      <!-- UUID: 30e1ed92-d1f8-40fb-b14c-9f90ff68eab4 -->
+      <!-- TARGET: arm-none-linux-gnueabi -->
+      <!-- SORT: glibc -->
+      <!-- ISSUE: CB-1356 -->
+      <formalpara components="license">
+        <title>VFP ABI dynamic linker compatibility</title>
+        <para>
+          VFP ABI sysroots included with &csl_prod; formerly included
+	  backward compatibility support for a change to the
+	  name of the dynamic linker in GLIBC version 2.15.  This
+	  transitional support has now been removed.  VFP ABI binaries
+	  compiled with GLIBC versions prior to 2.15 must be recompiled
+	  in order to run with the sysroots distributed with this
+	  version of &csl_prod;.
+        </para>
+      </formalpara>
+
       <!-- UUID: 0941c3aa-342a-4884-b463-c34005850845 -->
       <!-- TARGET: * -->
       <!-- SORT: glibc -->
-- 
2.8.1

